torchx/schedulers/ray/ray_driver.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import json
+import dataclasses
+import ray
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+import subprocess
+from .ray_common import RayActor
+
+@ray.remote
+class CommandActor:
+    def __init__(self, command: str, env: Dict[str, str]):
+        self.args = command.split(" ")
+        self.path = self.args[0]
+
+        # Set up the environment variables to be passed to the user's command.
+        self.env = os.environ.copy()
+        self.env.update(env)
+
+    def run_command(self) -> None:
+        os.execve(self.path, self.args, self.env)
+
+def load_actor_json(filename : str) -> List[RayActor]:
+    with open(filename) as f:
+        actors = []
+        for actor_dict in json.load(f):
+            actors.append(RayActor(**actor_dict))
+
+        return actors
+
+def _main(job_id = None):
+    print("Reading actor.json")
+    actors_dict = load_actor_json(str.startswith('actor'))
+    pgs = []
+    ray.init(address="auto", namespace="torchx-ray")
+
+    for actor_dict in actors_dict:
+
+        bundle = {"CPU": actor_dict["num_cpus"], "GPU": actor_dict["num_gpus"]}
```

Comment:
Since `load_actor_json` now returns `RayActor` instances, you should change the `actor_dict[]` expressions.